### PR TITLE
939 psss link to tupaia

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- Add preview tupaia button
- fix date picker regression bug
- fix storybook regression bug

---

### Screenshots:

Screen Shot 2020-09-09 at 6 05 08 PM

Screen Shot 2020-09-09 at 6 12 56 PM

### CODE REVIEW

File: packages/psss/src/components/Header.js [hunk 1]
```diff
@@ -129,14 +142,19 @@ export const Header = ({ Title, back, ExportModal }) => {
...
-            <>
-              <LightOutlinedButton onClick={() => setIsModalOpen(true)} startIcon={<SaveAlt />}>
-                Export Data
-              </LightOutlinedButton>
-              <ExportModal isOpen={isModalOpen} handleClose={() => setIsModalOpen(false)} />
-            </>
-          )}
+          <div>
+            <TupaiaButton href="https://tupaia.org" startIcon={<TupaiaIcon />}>
+              Preview Tupaia
```
Comment (edmofro) [inline packages/psss/src/components/Header.js:147]: Not sure about the wording "Preview". @enunan? It's more like "Visit" or "See" or "View"

**Review by edmofro — APPROVED** [review]
Just the wording question, but all the code looks good :-)

File: packages/psss/src/components/Header.js (hunk 1 above)
Comment (tcaiger) [reply packages/psss/src/components/Header.js:147]: Also another good question that @Klaus-S rasied. Should we open Tupaia.org in a new tab or just take them to it in the existing tab? It's Easy enough to implement either way.

Comment (edmofro) [reply packages/psss/src/components/Header.js:147]: New tab IMO

**Pull request merged**